test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/ReleaseBarnimIT.java

```diff
@@ -53,8 +53,8 @@ public void navigationSuccessful() throws Exception {
         assertEquals(36,
                 LogAssert.count(simulationRule, "Navigation.log", ".*Request to switch to new route for vehicle .*")
         );
-        // 14 adhoc vehicles + 12 cell vehicles
-        assertEquals(26,
+        // 14 adhoc vehicles + 10 cell vehicles
```

Comment:
We also have `ReleaseBarnimLibsumoIT` with the same asserts, they need to adjusted as well. This test is currently skipped in our CI pipeline, but should be successful if executed manually. I